src/module/actor/sheet/base.js

```diff
@@ -1350,3 +1389,93 @@ export class ActorSheetSFRPG extends ActorSheet {
         }
     }
 }
+
+// TODO(levirak): Wanted: better icons
+const iconLookupSkill = {
+    acr: 'icons/svg/d20.svg',
+    ath: 'icons/svg/d20.svg',
+    blu: 'icons/svg/d20.svg',
+    com: 'systems/sfrpg/icons/equipment/technological%20items/datapad.webp',
+    cul: 'icons/commodities/treasure/bust-carved-stone.webp',
+    dip: 'icons/skills/social/diplomacy-handshake.webp',
+    dis: 'icons/svg/d20.svg',
+    eng: 'systems/sfrpg/icons/equipment/goods/tool-manual.webp',
+    int: 'icons/svg/d20.svg',
+    lsc: 'icons/commodities/gems/gem-amber-insect-orange.webp',
+    med: 'systems/sfrpg/icons/equipment/technological%20items/medkit-advanced.webp',
+    mys: 'icons/magic/symbols/runes-triangle-magenta.webp',
+    per: 'icons/creatures/eyes/human-single-brown.webp',
+    phs: 'icons/tools/laboratory/vial-orange.webp',
+    pil: 'icons/svg/d20.svg',
+    pro: 'systems/sfrpg/icons/equipment/goods/credstick.webp',
+    sen: 'icons/svg/d20.svg',
+    sle: 'icons/svg/d20.svg',
+    ste: 'icons/svg/d20.svg',
+    sur: 'icons/tools/navigation/map-simple-tree.webp',
+};
+
+// TODO(levirak): Wanted: better icons
+const iconLookupSave = {
+    fort:   'icons/skills/social/intimidation-impressing.webp',
+    reflex: 'icons/skills/movement/figure-running-gray.webp',
+    will:   'icons/magic/holy/meditation-chi-focus-blue.webp',
+};
+
+// TODO(levirak): Wanted: better icons
+const iconLookupAbility = {
+    str: 'icons/magic/control/buff-strength-muscle-damage-orange.webp',
+    dex: 'icons/skills/movement/feet-winged-boots-brown.webp',
+    con: 'icons/svg/d20.svg',
+    int: 'icons/svg/d20.svg',
+    wis: 'icons/svg/d20.svg',
+    cha: 'icons/svg/d20.svg',
+};
+
+// TODO(levirak): where should this live?
+Hooks.on("hotbarDrop", async (bar, data, slot) => {
```

Comment:
This should probably live in the `item-macros.js` file, and that file should be renamed something more general, maybe just `macros` or `hotbar-macros` or something

Reply:
Thanks for the suggestion!